python/paddle/fluid/tests/unittests/test_translated_layer.py

```diff
@@ -94,7 +94,7 @@ def setUp(self):
             batch_size=BATCH_SIZE,
             shuffle=True,
             drop_last=True,
-            num_workers=2)
+            num_workers=0)
```

Comment:
Which UT will cover `num_workers=2`?

Reply:
test_multiprocess_dataloader_* related unittests

![image](https://user-images.githubusercontent.com/22561442/100421416-4cca1480-30c3-11eb-9538-935b1ec442e4.png)

need run exclusive, the multiprocess dataloader is not the test key point of TranslatedLayer